game/translations/tr.csv

```diff
@@ -0,0 +1,43 @@
+,de,en
```

Comment:
I would use the english text as key and have one file for every language

Reply:
Do you mean "About Suffragium" or "T_ABOUT_SUFFRAGIUM" as key.

This is how I would do it. https://github.com/b7g/Suffragium/pull/1
```
T_ABOUT_SUFFRAGIUM,Über Suffragium,About Suffragium
```
And maybe switch EN to the front an remove "T_".

Reply:
I merged the pr, so the keys are fixed.
But I don't think multiple files are a good idea, it just makes it more work to sync them.